agent/exploits/cve_2024_8963.py

```diff
@@ -0,0 +1,81 @@
+"""Agent Asteroid implementation for CVE-2024-8963"""
+
+import re
+import logging
+import datetime
+
+import requests
+from urllib import parse as urlparse
+
+from agent.exploits import webexploit
+from agent import exploits_registry
+from agent import definitions
+
+
+VULNERABILITY_TITLE = "Path Traversal in Ivanti CSA"
+VULNERABILITY_REFERENCE = "CVE-2024-8963"
+VULNERABILITY_DESCRIPTION = """Path Traversal in the Ivanti CSA before 4.6 Patch 519 allows a remote unauthenticated attacker to access restricted functionality."""
+RISK_RATING = "CRITICAL"
+PATCH_DATE = datetime.datetime(2024, 9, 10)
```

Comment:
Please document where this value came from.

Reply:
Done.